packages/mobile/android/app/build.gradle

```diff
@@ -73,8 +73,9 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    bundleAssetName: "index.android.bundle",
-    entryFile: "index.js"
+        bundleAssetName: "index.android.bundle",
```

Comment:
Is this option necessary, isn't "index.android.bundle" the default value?

Reply:
I think it's not necessary